webserver/static/robots.txt

```diff
@@ -2,3 +2,5 @@ User-agent: *
 Disallow: /current-status
 Disallow: /user/
 Disallow: /1/
+Disallow: /api
```

Comment:
Actually, I didn't know about robots.txt when I googled about it it said /filename /foldername/ so i thought robots.txt worked like that  but now i came to know that it blocks urls... I have changed the file now